java/src/jmri/InstanceManager.java

```diff
@@ -148,6 +148,23 @@
         return getDefault().getInstances(type);
     }
 
+    /**
+     * Retrieve a list of all objects of a specific type that were registered with
+     * {@link #store}.
+     *
+     * Intended for use with i.e. scripts where access to the class type is inconvenient.
+     * In Java code where typing is enforced, use {@link #getList(Class)}.
+     *
+     * @param className Fully qualified class name
+     * @return A list of type Objects registered with the manager or an empty
+     *         list.
+     */
+    @Nonnull
+    public static List<Object> getList(@Nonnull String className) {
+        return getDefault().getInstances(className);
```

Comment:
I think it would be good to add `@throws IllegalArgumentException` to the Javadoc for this method.
One possible interpretation of `getInstances()` is that it would return an empty list if the class name is wrong. By adding `@throws` to the Javadoc, it's clear what happens then a bad class name is given.